src/api/test-mocks/index.ts

```diff
@@ -87,3 +87,20 @@ export const mockRes = () => {
 
   return res;
 };
+
+export const INVALID_POSTCODES = {
+  ONE_LETTER_FIRST_PART: 'S2AA',
+  NO_LETTERS_FIRST_PART: '222AA',
+  TWO_DIGITS_SECOND_PART: 'SW122A',
+  THREE_DIGITS_SECOND_PART: 'SW1222',
+  ALL_LETTERS_SECOND_PART: 'SW1AAA',
+  TOO_MANY_CHARACTERS: 'SW1A2HQA',
+  TOO_FEW_CHARACTERS: 'S2AA',
+  TOO_MANY_CHARACTERS_WITHOUT_SPACE: 'SW1A2HQA',
+  TOO_FEW_CHARACTERS_WITHOUT_SPACE: 'S2AA',
+};
+
+export const VALID_POSTCODES = {
+  WITH_SPACE: 'SW1A 2HQ',
+  WITHOUT_SPACE: 'SW1A2HQ',
+};
```

Comment:
Could we move into their own file? Need to do for the others, files is becoming long